test_data.json

```diff
@@ -0,0 +1,191 @@
+[
```

Comment:
The format of this might change as we realize the need to add fields, but we should stabilize it pretty soon otherwise adding new fields will be a serious hassle as the file grows.